Client/Frontend/Browser/SearchEngines.swift

```diff
@@ -242,12 +280,15 @@ class SearchEngines {
 
     // Get all known search engines, possibly as ordered by the user.
     private func getOrderedEngines() -> [OpenSearchEngine] {
-        let unorderedEngines = SearchEngines.getUnorderedEngines()
+        var unorderedEngines = getCustomEngines()
+        unorderedEngines.appendContentsOf(SearchEngines.getUnorderedEngines())
```

Comment:
I think you can replace this with `let unorderedEngines = SearchEngines.getUnorderedEngines() + getCustomEngines()`